app/scripts/controllers/preferences.test.js

```diff
@@ -21,11 +21,11 @@ const NETWORK_CONFIGURATION_DATA = {
     rpcPrefs: {},
   },
 };
-describe('preferences controller', () => {
+describe('preferences controller', function () {
```

Comment:
Nit: accidental auto-formatter changes? I thought arrow functions were fine in Jest tests.

Reply:
Probably before my eslint restarted and realized it was a jest test now I'll fix.

Reply:
done